statistics/handle/handletest/statslock/stats_lcok_test.go

```diff
@@ -121,6 +121,22 @@ func TestStatsLockTableRepeatedly(t *testing.T) {
 	require.Nil(t, err)
 	tableLocked2 := handle.GetTableLockedAndClearForTest()
 	require.Equal(t, tableLocked1, tableLocked2)
+
+	// Unlock the table.
+	tk.MustExec("unlock stats t")
+	rows = tk.MustQuery("select count(*) from mysql.stats_table_locked").Rows()
+	num, _ = strconv.Atoi(rows[0][0].(string))
+	require.Equal(t, num, 0)
+
+	tk.MustExec("analyze table test.t")
+	tblStats2 := handle.GetTableStats(tbl.Meta())
+	require.Equal(t, int64(2), tblStats2.RealtimeCount)
+
+	// Unlock the table again and check the warning.
+	tk.MustExec("unlock stats t")
+	tk.MustQuery("show warnings").Sort().Check(testkit.Rows(
+		"Warning 1105 skip unlocking unlocked table: test.t",
+	))
```

Comment:
If result is one, it is unnecessary to sort it.

Reply:
Good catch! Addressed in [`32beb78` (#46478)](https://github.com/pingcap/tidb/pull/46478/commits/32beb785df4911cb8a8ac8515d58ce4091ce3e4a)